composer.json

```diff
@@ -5,14 +5,17 @@
     "license": [
         "MIT"
     ],
-    "repositories": [
```

Comment:
Was this done on purpose? Shouldn't we keep it?

Reply:
[There is no reason to ](https://getcomposer.org/doc/05-repositories.md#repository)

`Repositories are only available to the root package and the repositories defined in your dependencies will not be loaded. Read the [FAQ entry](https://getcomposer.org/doc/faqs/why-cant-composer-load-repositories-recursively.md) if you want to learn why.`